content/en/about/_index.md

```diff
@@ -16,5 +16,5 @@ using [Hugo](https://gohugo.io).
 [EGI Services](https://www.egi.eu/services/) is maintained by the EGI Community
 and coordinated by the
 [EGI Foundation](https://www.egi.eu/about/egi-foundation/). Everyone is invited
-to participate by following the [Contributing Guide](contributing).
+to participate, by following the [Contributing Guide](contributing).
```

Comment:
The comma is redundant, no need for it (confirmed by native English speaker in Slack on #english-language).

Reply:
Fixed